demo/checkbox-basic-demos.html

```diff
@@ -6,36 +6,131 @@
+    <vaadin-demo-snippet>
       <template preserve-content>
-        <vaadin-checkbox disabled checked>
-          Option two<br>
-          <small>Description for this option</small>
-        </vaadin-checkbox>
+        <vaadin-checkbox-group label="Disabled" theme="vertical" disabled>
+          <vaadin-checkbox value="1" checked>Option one</vaadin-checkbox>
+          <vaadin-checkbox value="2">Option two</vaadin-checkbox>
+          <vaadin-checkbox value="3">Option three</vaadin-checkbox>
+        </vaadin-checkbox-group>
 
-        <vaadin-checkbox disabled>Option two</vaadin-checkbox>
+        <vaadin-checkbox-group label="Disabled item" theme="vertical">
+          <vaadin-checkbox value="1">Option one</vaadin-checkbox>
+          <vaadin-checkbox value="2">Option two</vaadin-checkbox>
+          <vaadin-checkbox value="3" disabled>Option three</vaadin-checkbox>
+        </vaadin-checkbox-group>
+        <style>
+          vaadin-checkbox-group:first-of-type {
+            margin-right: 200px;
+          }
+        </style>
+      </template>
+    </vaadin-demo-snippet>
+
+    <h3>Object list</h3>
+    <vaadin-demo-snippet id="object-list" when-defined="vaadin-checkbox">
+      <template preserve-content>
+        <vaadin-checkbox-group label="Department" theme="vertical"></vaadin-checkbox-group>
+        <script>
+          window.addDemoReadyListener('#object-list', function(document) {
+            const departments = [
+              {id: '1', name: 'Product'},
+              {id: '2', name: 'Service'},
+              {id: '3', name: 'HR'},
+              {id: '4', name: 'Accounting'}
+            ];
+            const checkboxGroup = document.querySelector('vaadin-checkbox-group');
+            departments.forEach(function(item) {
+              const checkbox = window.document.createElement('vaadin-checkbox');
+              checkbox.textContent = item.name;
+              checkbox.value = item.id;
+              checkboxGroup.appendChild(checkbox);
+            });
+          });
+        </script>
+      </template>
+    </vaadin-demo-snippet>
+
+    <h3>Value change event</h3>
+    <vaadin-demo-snippet id="value-change-event" when-defined="vaadin-radio-button">
+      <template preserve-content>
+        <vaadin-checkbox-group label="Label" theme="vertical">
+          <vaadin-checkbox value="1">Option one</vaadin-checkbox>
+          <vaadin-checkbox value="2">Option two</vaadin-checkbox>
+          <vaadin-checkbox value="3">Option three</vaadin-checkbox>
+        </vaadin-checkbox-group>
+        <div id="output">Selected value:</div>
+        <script>
+          window.addDemoReadyListener('#value-change-event', function(document) {
+            const checkboxGroup = document.querySelector('vaadin-checkbox-group');
+            const output = document.querySelector('#output');
+            checkboxGroup.addEventListener('change', function(event) {
+              output.textContent = checkboxGroup.value;
```

Comment:
The output behaves a bit stange. First it says `Selected value:` but when you select items, it clears the text and just says `1,2`

Reply:
Done.

Reply:
It still works the same as before:
- First it says `Selected value:` (followed with a ":", indicating the selected values will be listed after it)
- Selecting values still replaces the "Selected value:" text with "Option two,Option three". I think the previous values "1,2" were better, but what it should say is "Selected value: 1, 2" (not clearing the "Selected value:" text).

Maybe it makes sense to have a separate container for the value. Something like `<div>Selected value: <span id="output"></span></div>`

Reply:
Done.